enterprise/frontend/src/metabase-enterprise/metabot/MetabotChat/MetabotChat.module.css

```diff
@@ -0,0 +1,48 @@
+.container {
```

Comment:
This might not work with other status messages, like `DatabaseStatus` (sync status) etc. We might need to align them in a single list